debian/ubuntu-advantage-tools.postinst

```diff
@@ -418,15 +418,24 @@ case "$1" in
         fi
       fi
 
-      # We modify permissions for ubuntu-advantage.log because
-      # in a past version of UA, this log file was world readable.
-      # This isn't necessary for the other log files,
-      # because they have always been only root-readable.
+      # log files need to be world-readable
       if [ ! -f /var/log/ubuntu-advantage.log ]; then
           touch /var/log/ubuntu-advantage.log
       fi
-      chmod 0600 /var/log/ubuntu-advantage.log
+      chmod 0644 /var/log/ubuntu-advantage.log
       chown root:root /var/log/ubuntu-advantage.log
+
+      if [ -f /var/log/ubuntu-advantage-timer.log ]; then
+          chmod 0644 /var/log/ubuntu-advantage-timer.log
+      fi
+
+      if [ -f /var/log/ubuntu-advantage-license-check.log ]; then
+          chmod 0644 /var/log/ubuntu-advantage-license-check.log
+      fi
+
+      if [ -f /var/log/ubuntu-advantage-daemon.log ]; then
+          chmod 0644 /var/log/ubuntu-advantage-daemon.log
+      fi
```

Comment:
I think we can use a loop structure here

Reply:
@CalvoM reminder that I think we want to make this small change before merging this. Something like:

```bash
for log_suffix in timer license-check daemon
do
    if [ -f /var/log/ubuntu-advantage-${log_suffix}.log ]; then
        chmod 0644 /var/log/ubuntu-advantage-${log_suffix}.log
    fi
done
```

CC @lucasmoura this is what you were imaging right?

Reply:
@orndorffgrant yes

Reply:
Okay, time to learn bash properly :-)